test/test_youtube_lists.py

```diff
@@ -66,6 +67,16 @@ def test_youtube_flat_playlist_titles(self):
         for entry in result['entries']:
             self.assertTrue(entry.get('title'))
 
+    def test_youtube_flat_playlist_uploaders(self):
+        dl = FakeYDL()
+        dl.params['extract_flat'] = True
+        ie = YoutubeTabIE(dl)
+        result = ie.extract('https://www.youtube.com/playlist?list=PL2zzFmCbIz4fshCLrvopue-wDeKQ5ERKO')
+        self.assertIsPlaylist(result)
+        for entry in result['entries']:
+            if entry['title'] not in ('[Deleted video]', '[Private video]'):
+                self.assertTrue(entry.get('uploader'))
```

Comment:
Test should be added to `_TESTS` of extractor class. Use one of the youtube-dl's test playlists for testing, e.g. https://www.youtube.com/playlist?list=PL4lCao7KL_QFVb7Iudeipvc2BCavECqzc.